lib/cisco_node_utils/bgp_af.rb

```diff
@@ -226,6 +226,22 @@ def default_additional_paths_selection
       config_get_default('bgp_af', 'additional_paths_selection')
     end
 
+    # advertise_l2vpn_evpn
```

Comment:
This property is confusing me - I don't see it in the cli even when I enable the features. I'd like a phone call to discuss this implementation.